### PR TITLE
Add a ZMQ perl exmple

### PR DESCRIPTION
**Pull request by inorton**
It took some sweat and tears but here is a zeromq client example that reads the EDDN feed!

### CODE REVIEW

File: examples/Perl5/client.pl [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/perl5
```
Comment (gedge) [inline examples/Perl5/client.pl]: `#!/usr/bin/env perl` is more portable

File: examples/Perl5/client.pl [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+use Compress::Zlib;
+
+my $endpoint = "tcp://eddn.edcd.io:9500";
+my $ctx      = ZMQ::FFI->new();
+
+my $s = $ctx->socket(ZMQ_SUB);
+
+$s->connect($endpoint);
+
+{
```
Comment (gedge) [inline examples/Perl5/client.pl]: is this nesting superfluous?

File: examples/Perl5/client.pl [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  $s->subscribe('');
+  while(1)
+  {
+    until ($s->has_pollin) {
```
Comment (gedge) [inline examples/Perl5/client.pl]: I'd be tempted to go idiomatic: `usleep 100_000 until $s->has_pollin;` here

File: examples/Perl5/client.pl [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      # compensate for slow subscriber
+      usleep 100_000;
+    }
+    my $data = $s->recv();
+
+    # the data is zlib compressed
+    my $compressed = scalar($data);
```
Comment (gedge) [inline examples/Perl5/client.pl]: unused?

File: examples/Perl5/client.pl [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+    # turn the json into a perl hash
+    my $pj = decode_json(uncompress($data));
+    my $schema = $pj->{"\$schemaRef"};
```
Comment (gedge) [inline examples/Perl5/client.pl]: `my $schema = $pj->{'$schemaRef'};` ftw

File: examples/Perl5/client.pl [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    printf STDERR "schema = %s\n", $schema;
```
Comment (gedge) [inline examples/Perl5/client.pl]: `warn "schema = $schema\n";`

Comment (gedge) [reply examples/Perl5/client.pl]: ditto for the other prints to STDERR

File: examples/Perl5/client.pl (hunk 2 above)
Comment (inorton) [reply examples/Perl5/client.pl]: Totally, I had no vimrc when writing this and indent totally went nuts when I tried to fix it :)

**Pull request merged**